css/main.css

```diff
@@ -325,3 +325,66 @@ li {
 .skills {
   list-style: inside url("../media/svg/circle.svg");
 }
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 4px 24px;
+}
+
+.form-container h1 {
+  width: 327px;
+  height: 363px;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  margin: 24px auto;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  margin: 38px auto;
+}
+
+.form-control input {
+  width: 327px;
+  padding: 15px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin: 12px auto;
+}
+
+.form-control textarea {
+  width: 327px;
+  height: 114px;
+  background: #fff;
+  box-sizing: border-box;
+  border-radius: 4px;
+  padding: 12px;
+  margin: 12px auto;
+  font-family: Inter, Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+  resize: none;
+}
+
+.form-button {
+  color: #fff;
+  background-color: #ff6b00;
+  padding: 15px;
+  width: 128px;
+  height: 48px;
+}
```

Comment:
[OPTIONAL] You can set  `border: none;` on your submit button or add a border that has the same color as the button.  Also, I think it would be better if you maybe increase the `font weight` of the button text. This will improve the look of your design to make it as close to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as possible.

Your button:
![image](https://user-images.githubusercontent.com/38088818/152544082-80af14b7-0d1a-409d-bb87-4034271ac831.png)

Figma Button:
![image](https://user-images.githubusercontent.com/38088818/152544279-d1ed20a2-f7f8-4080-bdd9-9ed25710af89.png)